README.md

```diff
@@ -152,6 +152,54 @@ Nap is customized through environment variables:
   />
 </p>
 
+## Snippet Management from Terminal
+- Emulate the functionality of cmd snippet managers like like pet(https://github.com/knqyf263/pet) and the-way(https://github.com/out-of-cheese-error/the-way)
```

Comment:
```suggestion
- Emulate the functionality of cmd snippet managers like [pet](https://github.com/knqyf263/pet) and [the-way](https://github.com/out-of-cheese-error/the-way)
```

Reply:
Hey thanks for the review @jolheiser. I have pushed the commit with the specified change as requested. Hope you find the changes good enough to be merged.Do let me know if any further changes are required